### ISSUE TITLE
PA Review: Tracks

### ISSUE THREAD

**Issue by Ian-MacLeod** [Ian-MacLeod/sound-macleod#4]
- [x] Backend: DB, model, controller, views
- [x] Redux Loop: ajax, actions, reducer
- [x] Presentational Components and Containers
- [x] Styling
- [x] Smooth, bug-free navigation
- [x] Adequate and appropriate seeds

**Comment by anastassia-b** [comment]
Nice progress! Excellent job on the styling and functionality so far. You've built out 3 views: the homepage, the track page, and the user page. 

Several bugs I'm experiencing:
* I am allowed to upload non-mp3 files as tracks. For example, I uploaded a .jpg. Is there a way to check that a file is an audio file before uploading, and display a notice to the user?
* When I uploaded a track, one time it uploaded twice. 
* When I delete my own track, the `REMOVE_TRACK` action is dispatched. After this the user profile breaks and doesn't render. The error is `Cannot read property 'id' of undefined`. Upon refresh, it did successfully delete the track. 
* I see a button to delete other's tracks. When I click it, it dispatches `REMOVE_TRACK` but instead plays it. The track is not deleted. Is there a way to only display the remove track button for your own songs? A bonus could be when I click the delete track button, a modal pops up that asks if I am _sure_ I want to delete it. 

Great job getting track creation and deletion functionality! In order to complete this MVP:
- [x] Address the above bugs 
- [x] Seed more tracks
- [x] Maybe style the track upload page to be less wide. And if it is centered, I think that'd look nicer.
